util/src/test/java/io/zeebe/util/StackdriverJSONLayoutTest.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.util;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectReader;
+import java.io.IOException;
+import java.io.StringWriter;
+import java.io.Writer;
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.logging.log4j.Level;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.apache.logging.log4j.core.Appender;
+import org.apache.logging.log4j.core.LoggerContext;
+import org.apache.logging.log4j.core.StringLayout;
+import org.apache.logging.log4j.core.appender.WriterAppender;
+import org.apache.logging.log4j.core.config.LoggerConfig;
+import org.junit.Before;
+import org.junit.Test;
+
+public class StackdriverJSONLayoutTest {
+
+  private Logger logger = LogManager.getLogger();
+  private LoggerContext context = (LoggerContext) LogManager.getContext(false);
+  private Writer logOutput;
+  private ObjectReader jsonReader = new ObjectMapper().reader();
+
+  @Before
+  public void before() {
+    logOutput = new StringWriter();
+  }
+
+  @Test
+  public void testJSONOutput() throws IOException {
+    // Given
+    createTestAppenderWithLayout(new StackdriverJSONLayout());
+    // when
+    logger.error("Should appear as JSON formatted ouput");
+    // then
+    final Map<String, String> jsonMap = logOutputToJSONMap();
+    assertThat(jsonMap)
+        .containsKeys(
+            "logger", "message", "severity", "thread", "timestampNanos", "timestampSeconds");
+    assertThat(jsonMap).containsEntry("message", "Should appear as JSON formatted ouput");
+    assertThat(jsonMap).containsEntry("logger", logger.getName());
+  }
+
+  private Map<String, String> logOutputToJSONMap() throws IOException {
+    return jsonReader
+        .withValueToUpdate(new HashMap<String, String>())
+        .readValue(logOutput.toString());
+  }
+
+  private void createTestAppenderWithLayout(StringLayout layout) {
+    final Appender appender =
+        WriterAppender.createAppender(layout, null, logOutput, "test", false, false);
+    final LoggerConfig loggerConfig = context.getConfiguration().getRootLogger();
+    loggerConfig.addAppender(appender, Level.ALL, null);
```

Comment:
Missing `loggerConfig.removeAppender(...)` as part of test cleanup. Or use a logger local to this test